sys/storage/nvs/nvs.c

```diff
@@ -28,18 +28,15 @@
  */
 
 
-#define ENABLE_DEBUG (1)
 #include "debug.h"
 #include "storage/nvs.h"
 #include "nvs_flash/include/nvs_flash.h"
 
+#define ENABLE_DEBUG (1)
 
 uint32_t m_handle;
 bool is_open = false;
```

Comment:
If you want you can remove this variable, if the NVS isn't initialized, all API calls will return an error (nvs not initialized), and thus, we'll return `-1` anyway.